about.html

```diff
@@ -0,0 +1,104 @@
+          </svg>
+        </a>
+      </li>
+      <li class="social-media"><a href="#">English</a></li>
+      <li class="social-media"><a href="#">MyPage</a></li>
+      <li class="social-media"><a href="#">Logout</a></li>
+    </ul>
+  </nav>
+  <nav class="navbar">
+    <button type="button" class="hamburger">
+      <svg xmlns="http://www.w3.org/2000/svg" width="60" height="60" fill="currentColor" class="bi bi-list"
+        viewBox="0 0 16 16">
+        <path fill-rule="evenodd"
+          d="M2.5 12a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5z" />
+      </svg>
+    </button>
+    <button type="button" class="close">
+      <svg xmlns="http://www.w3.org/2000/svg" width="60" height="60" fill="currentColor" class="bi bi-x-square" viewBox="0 0 16 16">
+        <path d="M14 1a1 1 0 0 1 1 1v12a1 1 0 0 1-1 1H2a1 1 0 0 1-1-1V2a1 1 0 0 1 1-1h12zM2 0a2 2 0 0 0-2 2v12a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2V2a2 2 0 0 0-2-2H2z"/>
+        <path d="M4.646 4.646a.5.5 0 0 1 .708 0L8 7.293l2.646-2.647a.5.5 0 0 1 .708.708L8.707 8l2.647 2.646a.5.5 0 0 1-.708.708L8 8.707l-2.646 2.647a.5.5 0 0 1-.708-.708L7.293 8 4.646 5.354a.5.5 0 0 1 0-.708z"/>
+      </svg>
+    </button>
+    <div class="desktop-title">
+      <a class="desktop-icon-link" href="index.html"><img class="desktop-icon" src="./assets/images/istockphoto-607972154-612x612.jpeg" alt="JazzMasters logo"></a>
+      <p class="desktop-name">JazzMasters</p>
+    </div>
+    <ul class="navbar-items">
+      <li class="navbar-item"><a class="navbar-links" href="index.html">Home</a></li>
+      <li class="navbar-item"><a class="navbar-links" href="#">Tickets</a></li>
+      <li class="navbar-item"><a class="navbar-links" href="#">Contact Us</a></li>
+    </ul>
+  </nav>
+
+  <main>
+    <h3 class="lede">"Be-bop, Re-bop, Doo-bop!"</h3>
+    <h1 class="title">JAZZ MASTERS</h1>
+    <article>
+      <p class="main-description">
+        The first edition of the JazzMasters concert is going to allow thousands of fans from all over the globe to
+        watch some previously
+        unreleased footage from some of our dearest heroes.
+      </p>
+    </article>
+    <article>
+      <p class="main-tag">
+        Please contact us per email for any further questions about JazzMaster!
+      </p>
+      <p class="email-address">jazzmasters@jazzmasters.com</p>
+    </article>
+  </main>
+
+  <section class="logo">
+    <h2 class="logo-title">JazzMasters Logo</h2>
+    <p class="logo-description">The logo for the first edition of the JazzMasters concert was decided through the
+      international JazzMasters Logo competition.</p>
+    <img class="logo-image" src="./assets/images/istockphoto-607972154-612x612.jpeg" alt="The JazzMasters logo">
+  </section>
```

Comment:
- [ ] Please note that the about page should contain a bottom section with a centred column that includes a section header, a divider, a line of text, and two images with text centred over them😉.
Like this👇🏽
![behance](https://user-images.githubusercontent.com/44624138/142332487-f4a80e2c-7709-44ea-9194-9f17635bf7ce.jpg)

Reply:
Please note that on mobile the bottom section images are organized in a vertical column👌 and on desktop, the bottom section images are organized in a horizontal row😉. [See design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)